README.md

```diff
@@ -31,81 +31,255 @@ yarn add react-native-fs2
 await RNFS.mkdir(`FolderToCreate`);
-// mkdir(filepath: string, options?: MkdirOptions): Promise<undefined>;
 ```
 
  * Creates directory at `filepath` location.
  * Optionally include `MkdirOptions` with properties:
    * (iOS) - [NSURLIsExcludedFromBackupKey](https://developer.apple.com/documentation/foundation/nsurlisexcludedfrombackupkey)
    * (iOS) - [NSFileProtectionKey](https://developer.apple.com/documentation/foundation/nsfileprotectionkey)
 
-#### moveFile
+### `moveFile`
 ```ts
+// moveFile(filepath: string, destPath: string, options?: FileOptions): Promise<undefined>
 await RNFS.moveFile('FileToMove', 'DestinationLocation')
-// moveFile(filepath: string, destPath: string, options?: FileOptions): Promise<undefined>;
 ```
 
 * Moves file from `filepath` to `destPath`
 * Optionally includes `FileOptions` with properties:
   * (iOS) - [NSFileProtectionKey](https://developer.apple.com/documentation/foundation/nsfileprotectionkey)
 
-#### copyFile
+### `copyFile`
 ```ts
+// copyFile(filepath: string, destPath: string, options?: FileOptions): Promise<undefined>
 await RNFS.copyFile('FileToCopy', 'DestinationLocation')
-// copyFile(filepath: string, destPath: string, options?: FileOptions): Promise<undefined>;
 ```
 
 * Copies file from `filepath` to `destPath`
 * Optionally includes `FileOptions` with properties:
   * (iOS) - [NSFileProtectionKey](https://developer.apple.com/documentation/foundation/nsfileprotectionkey)
 
-#### getFSInfo
+### `getFSInfo`
+```ts
+// getFSInfo(): Promise<FSInfoResult>
+const fsInfo = await RNFS.getFSInfo()
+```
+* Returns an `FSInfoResult` object that contains information on the device storage space
+* `FSInfoResult`
+  * totalSpace: `number` -> The total amount of storage space on the device (in bytes).
+  * freeSpace: `number` -> The amount of available storage space on the device (in bytes)
 
-#### getAllExternalFilesDirs
+### `getAllExternalFilesDirs` (Android only)
+```ts
+// getAllExternalFilesDirs(): Promise<string[]>
+const externalFileDirs = await RNFS.getAllExternalFilesDirs()
+```
+* Returns an `array` with the absolute paths to application-specific directories on all shared/external storage devices where the application can place persistent files it owns.
 
-#### unlink
+### `unlink`
+```ts
+// unlink(filepath: string): Promise<void>
+await RNFS.unlink('FileToUnlink')
+```
+* Unlinks the item at `filepath`. If the item does not exist, an error will be thrown.
+Also recursively deletes directories (works like Linux `rm -rf`).
 
-#### exists
+### `exists`
+```ts
+// exists(filepath: string): Promise<boolean>
+await RNFS.exists('File')
+```
+* Check if the item exists at `filepath`. If the item does not exist, return false.
 
-#### stopDownload
 
-#### resumeDownload
+### `stopDownload`
+```ts
+// stopDownload(jobId: number): void
+await RNFS.stopDownload('JobID'): void
+```
+* Abort the current download job with this ID. The partial file will remain on the filesystem.
```

Comment:
should we position this along with `resumeDownload` and `isResumable` next to the `downloadFile` documentation? that way users reading the `downloadFile` documentation will able to see all download related methods available in one group